pages/advanced-algorithms.mdx

```diff
@@ -11,11 +11,13 @@ If you require procedures designed to solve specific graph problems, there is a
 number of advanced algorithms available in Memgraph.
 
 [BFS](/advanced-algorithms/built-in-graph-algorithms#breadth-first-search),
-[DFS](/advanced-algorithms/built-in-graph-algorithms#depth-first-search), [Weighted shortest
+[DFS](/advanced-algorithms/built-in-graph-algorithms#depth-first-search),
+[Weighted shortest
 path](/advanced-algorithms/built-in-graph-algorithms#weighted-shortest-path),
 [All shortest
 paths](/advanced-algorithms/built-in-graph-algorithms#all-shortest-paths) are
-built-in algorithms you can run using their specific clauses. 
+built-in Deep path traversal algorithms you can run using their specific
```

Comment:
If used within a sentence, I think there is no need for uppercase (Deep) -> either way, align to be consistent across docs